### ISSUE TITLE
TextWire float64 loses precision for values of form E-4

### ISSUE THREAD

**Issue by gregallen** [OpenHFT/Chronicle-Wire#104]
Seems to have only 11 digits of precision - should be nearer 16

```
    @Test
    public void shouldNotLoseDoublePrecisionOverTextWire()
    {
        final Bytes<?> bytes = Wires.acquireBytes();

        final Wire wire = WireType.TEXT.apply(bytes);
        final double d = 0.000212345678901;
        wire.getValueOut().float64(d);

        final TextWire textWire = TextWire.from(bytes.toString());
        final double d2 = textWire.getValueIn().float64();

        assertThat(d2, is(d));
    }
```

**Comment by gregallen** [comment]
any chance of a fix for this?

**Comment by hft-team-city** [comment]
Released in Chronicle-Wire-2.17.5 (https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.17.5), BOM-2.17.30 (https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.30)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
